### PR TITLE
Update validator owners

### PR DESCRIPTION
**Pull request by honeybadgerdontcare**
Require validator owners to approve changes.

### CODE REVIEW

**Comment by kristoferbaxter** [comment]
Curious why this would be different than other segments of the amphtml codebase.

Right now src/ level owners can approve changes to this directory and with this PR that would be lost.

File: validator/OWNERS [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{name: 'ampproject/wg-caching', notify: true}],
+      owners: [{name: 'ampproject/wg-caching', require: true}],
```
Comment (rcebulko) [inline validator/OWNERS:7]:
```suggestion
      owners: [{name: 'ampproject/wg-caching', required: true}],
```

image

**Comment by rcebulko** [comment]
> Curious why this would be different than other segments of the amphtml codebase.
> 
> Right now src/ level owners can approve changes to this directory and with this PR that would be lost.

I don't believe this is accurate. Owners in `amphtml/src/OWNERS` can approve anything within `src/`. Top-level Owners in `amphtml/OWNERS` can approve changes in `amphtml/validator`, but there are only three: Malte, Dima, and Justin.

**Comment by kristoferbaxter** [comment]
> I don't believe this is accurate. Owners in amphtml/src/OWNERS can approve anything within src/. Top-level Owners in amphtml/OWNERS can approve changes in amphtml/validator, but there are only three: Malte, Dima, and Justin. 

Yes, the intention with my comment was top-level owners, but I didn't specify the correct path.

> The effect of this change would be that, in the case of a PR that one of those three approved (such as Justin, in the case of the PR that prompted this one), approval from someone in wg-caching would still be required.

I'm questioning if this segment of the codebase requires this difference versus the remainder. Top level owners are rare and are expected to have the knowledge necessary to approve all work in the codebase.

**Comment by rcebulko** [comment]
> I'm questioning if this segment of the codebase requires this difference versus the remainder. Top level owners are rare and are expected to have the knowledge necessary to approve all work in the codebase.

Will leave it to @honeybadgerdontcare to make that assessment, but to add a sprinkle of extra context, users with admin positions on the repo (which I assume the Big Three have) are able to override the Owners check to force merge in the event of something like a major build breakage. So it doesn't fully block anyone out.

File: validator/OWNERS (hunk 1 above)
Comment (rcebulko) [reply validator/OWNERS]: Update: I committed this suggested change directly though the UI to save you one round-trip checking the thread, updating, and waiting for presubmits

**Comment by honeybadgerdontcare** [comment]
1. specific domain knowledge is useful here (who should allow validation changes)
2. having rules and engine in alignment is useful here (release coordination may matter)
3. as pointed out there are overrides if necessary (this is not a project blocker)

It's easy to make something valid and difficult to make something valid invalid. I personally would like to be sure that anything made valid is reviewed by someone with domain knowledge.

**Review by kristoferbaxter — APPROVED** [review]

**Pull request merged**